display_list/display_list_matrix_clip_tracker.cc

```diff
@@ -323,7 +336,9 @@ SkRect Data3x3::local_cull_rect() const {
     // cull rect.
     return DisplayListBuilder::kMaxCullRect;
   }
-  return inverse.mapRect(cull_rect_);
+  SkRect expended_rect;
```

Comment:
What about the version in Data4x4?

Reply:
This is dubious, btw, because we could be computing local bounds for something that won't yet have its final transform, so the roundOut may be too much or not enough depending on a scale that happens outside the picture recording...

Reply:
There is roundOut logic in DLBuilder. If it is not modified here, some tests in display_list_unittests.cc and canvas_test.dart will fail.

So we should change the tests instead of here, right?